go/vt/vtctl/workflow/traffic_switcher.go

```diff
@@ -440,21 +440,18 @@ func (ts *trafficSwitcher) deleteShardRoutingRules(ctx context.Context) error {
 }
 
 func (ts *trafficSwitcher) deleteKeyspaceRoutingRules(ctx context.Context) error {
-	log.Infof("deleteKeyspaceRoutingRules: workflow %s.%s", ts.targetKeyspace, ts.workflow)
 	if !ts.IsMultiTenantMigration() {
 		return nil
 	}
-	krr, err := topotools.GetKeyspaceRoutingRules(ctx, ts.TopoServer())
-	if err != nil {
-		return err
-	}
-	for _, suffix := range tabletTypeSuffixes {
-		delete(krr, ts.SourceKeyspaceName()+suffix)
-	}
-	if err := topotools.SaveKeyspaceRoutingRules(ctx, ts.TopoServer(), krr); err != nil {
-		return err
-	}
-	return nil
+	log.Infof("deleteKeyspaceRoutingRules: workflow %s.%s", ts.targetKeyspace, ts.workflow)
+	name := fmt.Sprintf("Deleting %s", ts.SourceKeyspaceName())
```

Comment:
This is the reason rather than name, no? It would also be more descriptive IMO to say: `Deleting rules for %s` there.